cloudformation/batch-substack.cfn.json

```diff
@@ -418,8 +418,6 @@
     "ManageDockerImagesCustomResource": {
       "Type": "AWS::CloudFormation::CustomResource",
       "DependsOn": [
-        "CodeBuildDockerImageBuilderProject",
```

Comment:
Why are we removing these dependencies?

Reply:
Cause when the resource is already referenced in another field it is unneeded to explicitly add it again in the depends on block.